drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/FieldItemViewImpl.html

```diff
@@ -15,7 +15,9 @@
   ~ limitations under the License.
   -->
 
-<li data-field="fieldElement">
+<li data-field="fieldElement" class="kie-list-view-pf-main-info__check">
+    <span data-field="fieldNameElement"></span>
+    <span data-field="checkElement" class="pull-right fa fa-check" style="padding-top: 4px; display: none"></span>
```

Comment:
Is it possible to move these `style` values to CSS?

Reply:
Yes, it is. Done!